.github/mergify.yml

```diff
@@ -139,13 +139,13 @@ pull_request_rules:
           - branch-perf-v15
         assignees:
           - "{{ author }}"
-  - name: Automate backport pull request manager-3.2
+  - name: Automate backport pull request manager-3.3
```

Comment:
Why replacing and not appending? 

You do not plan to have any backports to manager-3.2 anymore?

Reply:
Sorry, my bad, we still might need it.
Thanks for bringing this to my attention.
PR has been updated.